standalone-metastore/src/main/sql/mssql/upgrade-2.3.0-to-3.0.0.mssql.sql

```diff
@@ -210,8 +210,30 @@ ALTER TABLE COMPLETED_TXN_COMPONENTS ADD CTC_WRITEID bigint;
   -- add a new column to support default value for DEFAULT constraint
  ALTER TABLE KEY_CONSTRAINTS ADD DEFAULT_VALUE VARCHAR(400);
 
-ALTER TABLE HIVE_LOCKS ALTER COLUMN HL_TXNID bigint NOT NULL;
+ALTER TABLE HIVE_LOCKS MODIFY ALTER COLUMN HL_TXNID bigint NOT NULL;
```

Comment:
Looks you modify the command in master code. Is it intentional or overwritten by mistake?

Reply:
reverted ..i think merge issue